src/app/models/course.model.ts

```diff
@@ -1,9 +1,8 @@
-import { Time } from '@angular/common';
-
 export interface Course {
   id: string;
   title: string;
   creationDate: Date;
-  duration: Time;
+  duration: number;
   description: string;
+  topRated: boolean;
```

Comment:
you can make it optional. In that case you can skip declaring false everywhere)

```
 topRated?: boolean;

Reply:
made optional